bot/cogs/snekbox.py

```diff
@@ -43,6 +46,25 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.jobs = {}
 
+        # Dict[int[user_id],
+        #      Tuple[str[saved_code],
+        #            datetime[last_used]]]
+        self.contexts = {}
+        self._run = True
+
+        async def clean_task():
```

Comment:
Linter: `TYP201 Missing return type annotation for public function`

Also where's the docstring here.